node-commons/src/main/java/org/apache/iotdb/commons/utils/ThriftCommonsSerDeUtils.java

```diff
@@ -278,4 +280,13 @@ public static TSchemaNode deserializeTSchemaNode(ByteBuffer buffer) {
     }
     return schemaNode;
   }
+
+  private static TByteBuffer generateTByteBuffer(ByteBuffer buffer) throws TTransportException {
+    // TByteBuffer uses the default TConfiguration which limits the MaxMessageSize
+    // we use ByteBuffer.slice() to update the origin buffer if buffer.capacity() >= MaxMessageSize
+    if (buffer.capacity() >= THRIFT_DEFAULT_MAX_MESSAGE_SIZE) {
+      buffer.slice();
```

Comment:
It looks like this function doesn't do anything?
<img width="582" alt="image" src="https://user-images.githubusercontent.com/32640567/222107424-f2b33317-673b-4502-9612-25d2ef4f4215.png">

Reply:
update the  capacity